common/src/main/java/com/wynntils/wynn/handleditems/properties/CountedItemProperty.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.handleditems.properties;
+
+public interface CountedItemProperty {
```

Comment:
Item properties could be functional interfaces?

Reply:
I assume you mean "tagged `@FunctionalInterface`". They already *are* functional interfaces, since by definition any interface with a single method is a functional interface.

However, that is mostly by chance. There's nothing inherently wrong if any of these would actually need two methods, perhaps a `String getUnit()` as well. That indicates that they are not functional interfaces by "heart". In contrast with e.g. `Runnable` which would be *pointless* if it had another method. 

So no, I don't think they should be tagged `@FunctionalInterface`.